Secure-Software-Development-Policy/Zowe-SSDP-Organization/Policy.md

```diff
@@ -0,0 +1,206 @@
+
+```TODO: The organization/contributors developing Zowe component, should either reuse the tools maintained by Zowe or shall use own tools adequate to the once provided by Zowe and providing consistent results to satisfy security criteria defined in both ZSSD policies.```     
+
+- A2.2:Deploy and Maintain Tools (#ID: ZSSD-OP:PO-ST-DMT)\
+```#REF: SSDF:PO.3.2```
+```#KEEP```
+
+Follow recommended security practices deploying and maintaining tools and toolchains.
+
+```TODO: consider removing or mergin with the previous one or be flexible enough to have an option for different kinds of contributors :individual, squad, company with own tool chains?```
+
+- A2.3 Generate Evidence and Artifacts (#ID: ZSSD-OP:PO-ST-GEA)\
+```#REF: SSDF:PO.3.3```\
+```#KEEP```
+
+Configure tools to generate evidence and artifacts of their support of secure software development practices as defined by Zowe.
+
+#### A3. Security Criteria (#ID: ZSSD-OP:PO-SC) 
+```#REF:  SSFD:PO.4 Define and Use Criteria for Software Security Checks)```\
+```#KEEP```
+
+Provide some kind of automatic validation for the Zowe Secure Development criteria - validate during pipeline execution.
+
+```#Q: Should we require projects to provide own checks for the criteria they declare?```\
+```How to ensure, enforce?```\
+```Code level security checks.```\
+```Dependency level security checks.```\
+```SonarCloud is already used and has some security checks.```\
+
+```TODO: Rephrase - maintain some criteria at Zowe level, while allowing projects to have additional specific criteria.```
+
+- A3.1 Software Security Checks (#ID: ZSSD-OP:PO-SC-SSC)
+```#REF: SSDF:PO.4.1```
+```#KEEP```  
+
+Define criteria for software security checks and track them throughout the SDLC.
+
+- A3.2 Safeguard the Necessary Information (#ID: ZSSD-OP:PO-SC-SNI):
+``` #REF: SSDF:PO.4.2```\
+```#KEEP```
+```#TOO-BROAD```
+
+Implement processes, mechanisms, etc. to gather and safeguard the necessary information in support of the criteria.
+
+#### A4. Secure Environments (#ID: ZSSD-OP:PO-SE)
+```#REF: SSFD:PO.5 Implement and Maintain Secure Environments for Software Development```
+
+- A4.1 Separate and Protect the Environments (#ID: ZSSD-OP:PO-SE-SPE):\
+```#REF: SSDF:PO.5.1```
+
+Separate and protect each environment involved in software development.
+
+```#Q: What is Dev and Test in the Zowe understanding? Are the pipelines on separate environments?```\  
+```#Q: Do we have a Prod at all?```\
+```#Q Do we mix environments in any sense?```s
+
+
+- A4.2 Harden Development Endpoints (#ID: ZSSD-OP:PO-SE-HDE)\
+```#REF: SSDF:PO.5.2```
+
+Secure and harden development endpoints (i.e., endpoints for software designers, developers, testers, builders, etc.\) to perform development-related tasks using a risk-based approach.
+
+```#Q: Can we require and enforce that? From organizations like IBM, BC and Rocket probably yes, but individuals and smaller teams may just not have resources or the will for that.```  
```

Comment:
Public infrastructure security is measurable. personal development machines such as my laptop is hard to measure! If I'm an independent contributor, I may not have a good way to prove this item. If I work at an organization, I may still have issue in providing the proof even if it exists internally.